pkgs/games/super-tux-kart/default.nix

```diff
@@ -1,45 +1,53 @@
-{ fetchgit, fetchsvn, cmake, stdenv, plib, SDL, openal, freealut, mesa
-, libvorbis, libogg, gettext, libXxf86vm, curl, pkgconfig
-, fribidi, autoconf, automake, libtool, bluez, libjpeg, libpng }:
+{ stdenv, fetchFromGitHub, fetchsvn, cmake, pkgconfig
+, openal, freealut, mesa
+, libvorbis, libogg, gettext, curl, pkgconfig, freetype
```

Comment:
You're using pkgconfig twice here. This doesn't evaluate: 

    error: duplicate formal function argument ‘pkgconfig’ at /home/fpletz/src/nixpkgs/pkgs/games/super-tux-kart/default.nix:1:45